Context LSP Log Panel.sublime-menu

```diff
@@ -1,7 +1,12 @@
 [
     {
+        "caption": "Clear Log Panel",
         "command": "lsp_clear_log_panel",
-        "caption": "Clear Log Panel"
+    },
+    {
+        "caption": "Limit to 500 Lines",
```

Comment:
```suggestion
        "caption": "Limit To 500 Lines",
```

Reply:
My suggested change looks weird, but it is consistent.

Reply:
I was thinking of suggesting `"Limit Logs To 500 Lines"`, but the word `Logs` is maybe unnecessary.

Reply:
Here is excerpt from some random site (https://capitalizemytitle.com/):

While the above words are generally capitalized in titles regardless of style, there are some words that are generally not capitalized when using title case. Again, these will depend on the specific style you choose (see [Title Capitalization Rules by Style](https://capitalizemytitle.com/#capitalizationrules) section). These include short words and conjunctions:

- Articles (a, an, the)
- Coordinating Conjunctions (and, but, for)
- Short (fewer than 4 letters)
- [Prepositions](https://capitalizemytitle.com/what-is-a-preposition/) (at, by, to, etc.)

Reply:
If you look through ST menu items you'll see that this is often followed but maybe not 100% consistently.

Reply:
> I was thinking of suggesting `"Limit Logs To 500 Lines"`, but the word `Logs` is maybe unnecessary.

It's a context menu for the log panel so it would be redundant.

(And to be fair we could also change the "Clear Log Panel" to just "Clear")

Reply:
Ok, resolved, there is a convention for that. It does look weird when it is uppercase.
 
+1 for "Clear"